src/components/ReadingGoalPage/ReadingGoalTargetAmountTab.tsx

```diff
@@ -323,7 +34,21 @@ const ReadingGoalTargetAmountTab: React.FC<ReadingGoalTabProps> = ({ state, disp
         <p className={styles.subtitle}>{t('goal-target.description')}</p>
       </div>
       <div className={styles.optionsContainer}>
-        {getInput()}
+        <ReadingGoalInput
+          type={type}
+          pages={pages}
+          seconds={seconds}
+          rangeStartVerse={rangeStartVerse}
+          rangeEndVerse={rangeEndVerse}
+          onRangeChange={(newRange) => dispatch({ type: 'SET_RANGE', payload: newRange })}
+          onPagesChange={(newPages) =>
```

Comment:
I see there is an inconsistency with passing functions as props inline vs separate functions.